app/models/school.rb

```diff
@@ -2,6 +2,8 @@
 
 class School < ApplicationRecord
   belongs_to :network, optional: true
+  belongs_to :local_authority_district, optional: true
```

Comment:
I think this is going to cause issues. Schools can change LADs (@cfq was it Nottinghamshire? Northamptonshire? You said somewhere was having a reshuffle). That's going to really mess up our historical billing for sparsity if we aren't careful

Reply:
Added a todo to deal with it. Don't think we need to worry about it just yet.